components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentsSerializerTest.kt

```diff
@@ -100,53 +130,79 @@ class ExperimentsSerializerTest {
     @Test
     fun toJsonValid() {
         val experiments = listOf(
-            Experiment("experiment-id",
-                "first",
-                "Description",
-                Experiment.Matcher("eng|es|deu|fra", "^org.mozilla.firefox_beta${'$'}", listOf("esp")),
-                Experiment.Bucket(100, 0),
-                1523549895713),
-            Experiment("experiment-2-id",
-                "second",
-                "SecondDescription",
-                Experiment.Matcher("es|deu", "^org.mozilla.firefox${'$'}", listOf("deu")),
-                Experiment.Bucket(10, 5),
-                1523549895749)
+            testDefaultExperiment(
+                id = "experiment-1-id",
+                description = "Description",
+                match = testDefaultMatcher(
+                    localeLanguage = "eng|es|deu|fra",
+                    appId = "^org.mozilla.firefox_beta${'$'}",
+                    regions = listOf("esp")
+                ),
+                buckets = Experiment.Buckets(0, 100),
+                branches = emptyList(),
+                lastModified = 1523549895713
```

Comment:
nit: magic number (what date is this and is it significant for someone to know it)?